crates/eframe/src/epi.rs

```diff
@@ -690,6 +695,15 @@ impl Frame {
         self.gl.as_ref()
     }
 
+    /// Register your own [`glow::Texture`],
+    /// and then you can use the returned [`egui::TextureId`] to render your texture with [`egui`].
+    ///
+    /// This function will take the ownership of your [`glow::Texture`], so please do not delete your [`glow::Texture`] after registering.
+    #[cfg(all(feature = "glow", not(target_arch = "wasm32")))]
+    pub fn register_native_texture(&mut self, native: glow::Texture) -> egui::TextureId {
```

Comment:
let's avoid name clashes in case we add a similar function for wgpu

```suggestion
    pub fn register_native_glow_texture(&mut self, native: glow::Texture) -> egui::TextureId {
```